Fails to build with Qt6 (-DENABLE_QT6=ON)

Regressed by #9668 due to qt/qtbase@c9b2528a9d60. From [error log](https://github.com/yuzu-emu/yuzu/files/10504625/yuzu-qt6-s20230125.log):
```c++
src/yuzu/main.cpp:4458:40: error: no member named 'AA_DisableWindowContextHelpButton' in namespace 'Qt'
    QCoreApplication::setAttribute(Qt::AA_DisableWindowContextHelpButton);
                                   ~~~~^
```

--- a/src/yuzu/main.cpp
+++ b/src/yuzu/main.cpp
@@ -4455,8 +4455,10 @@ int main(int argc, char* argv[]) {
     }
 #endif
 
-    // Disables the "?" button on all dialogs.
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+    // Disables the "?" button on all dialogs. Disabled by default on Qt6.
     QCoreApplication::setAttribute(Qt::AA_DisableWindowContextHelpButton);
+#endif
 
     // Enables the core to make the qt created contexts current on std::threads
     QCoreApplication::setAttribute(Qt::AA_DontCheckOpenGLContextThreadAffinity);

